docs/src/pages/components/tables/EnhancedTable.js

```diff
@@ -154,7 +154,7 @@ function EnhancedTableHead(props) {
             key={headCell.id}
             align={headCell.numeric ? 'right' : 'left'}
             padding={headCell.disablePadding ? 'none' : 'default'}
-            sortDirection={orderBy === headCell.id ? order : false}
+            sortDirection={orderBy === headCell.id && order}
```

Comment:
How does this change help the demo?

Reply:
The demo looks great. I flagged the unneeded ternary in case others agreed.

https://eslint.org/docs/rules/no-unneeded-ternary

Reply:
`no-unneeded-ternary` is not even enabled in the airbnb config we're using

> You can turn this rule off if you are not concerned with unnecessary complexity in conditional expressions.

Maybe "complexity" is lost in translation here but I find the ternary here "less complex". At least for humans since you don't have to know how boolean expressions are evaluated in TypeScript. It's entirely possible that some language always evaluates `a && b` expressions to `true` or `false` and never to their uncasted value.